File: Two Sum/TwoSum.java
import java.util.*;

class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map <Integer,Integer> result = new HashMap<>();
        for (int i = 0; i < nums.length; i++) {
            int b = target - nums[i];
            if (result.containsKey(b)) {
               return new int[] {result.get(b),i};
            }else{
                result.put(nums[i], i);
            }
        }
        return new int[] {};
    }
}

File: Two Sum/two_sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        result = {}
        for i in range(len(nums)):
            b = target - nums[i]
            if b in result:
                return [result[b], i]
            else:
                result[nums[i]] = i